manifests/params.pp

```diff
@@ -221,7 +221,8 @@
   $tftp_listen_on   = 'https'
   $tftp_managed     = true
   $tftp_manage_wget = true
-  $tftp_dirs        = ["${tftp_root}/pxelinux.cfg","${tftp_root}/boot","${tftp_root}/ztp.cfg","${tftp_root}/poap.cfg"]
+  # TODO - this will require migration for tftp_dirs YAML
```

Comment:
maybe the migration will be to just delete it, so it gets set the defaults again ;)
otherwise appending the new entires would also be fine of course